src/robot/libraries/BuiltIn.py

```diff
@@ -613,9 +613,17 @@ def should_be_equal(self, first, second, msg=None, values=True):
         for example, string ``false`` or ``no values``. See `Boolean arguments`
         section for more details.
 
+        ``ignore_case`` is False by default.  If True, it indicates that
+        ``first`` and ``second`` should be compared case-insensitively,
+        provided that ``first`` and ``second`` are string types.  See `Boolean
+        arguments` section for more details.  (New in RF 3.0.1)
+
         If both arguments are multiline strings, the comparison is done using
         `multiline string comparisons`.
         """
+        if is_truthy(ignore_case) and is_string(first) and is_string(second):
+            first = first.lower()
+            second = second.lower()
```

Comment:
So, are you advocating for the stance that users of IronPython 2.7.x and below should upgrade to fix the bug with "string.lower()" in their interpreter  or modify all the "string.lower()" calls to "utils.lower(string)" to workaround that bug?
My opinion says to leave the string conversions and have the users upgrade since bugs with "string.lower()".